config/database.yml

```diff
@@ -7,7 +7,7 @@ default: &default
 
 development:
   <<: *default
-  database: goodcity_server_development
+  database: from_staging
```

Comment:
oops... can revert this!

Reply:
whoops

Reply:
Should not have used a git gui. Sticking to console from now on